libraries/kotlinlibrary/build.gradle

```diff
@@ -16,3 +16,10 @@ allOpen {
   annotations("demo.Anno")
   annotations("demo.Anno2")
 }
+
+compileKotlin {
```

Comment:
can we put these at top level build.gradle?

Reply:
No, these come from the kotlin plugin when it's applied. Unless you mean can we DRY this up in a subprojects iteration at root, in which case - I thought about it, but figured these should be as much simple examples as they are functional tests